WASM fails to compile due to ginkgo dsl

i'm seeing two errors atm I'll open a PR for the first one shortly but ginkgo shouldn't prevent compilation. 
```
vendor/github.com/onsi/ginkgo/v2/internal/progress_report.go:26:9: undefined: PROGRESS_SIGNALS
vendor/github.com/onsi/ginkgo/v2/core_dsl.go:272:33: undefined: internal.NewOutputInterceptor
```

--- a/internal/output_interceptor_wasm.go
+++ b/internal/output_interceptor_wasm.go
@@ -1,0 +1,7 @@
+//go:build wasm
+
+package internal
+
+func NewOutputInterceptor() OutputInterceptor {
+	return &NoopOutputInterceptor{}
+}

--- a/internal/progress_report_wasm.go
+++ b/internal/progress_report_wasm.go
@@ -1,0 +1,10 @@
+//go:build wasm
+
+package internal
+
+import (
+	"os"
+	"syscall"
+)
+
+var PROGRESS_SIGNALS = []os.Signal{syscall.SIGUSR1}

